dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -935,4 +935,51 @@ public Results<NotFound, Ok<string>> Get()
     var result = GetResult();
     return result == null ? TypedResults.NotFound() : TypedResults.Ok(result);
 }
+```
+
+## ACL1018 - Code analysis suppression attribute requires Justification
+
+<table>
+<tr>
+    <td>Category:</td>
+    <td>Maintainability</td>
+</tr>
+<tr>
+    <td>Audacia coding standard:</td>
+    <td>N/A</td>
+</tr>
+</table>
+
+ACL1018 is based on the StyleCopAnalyzers Rules [SA1071](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1404.md). Which checks that the `SuppressMessage`, `MaxMethodLength` and `MaxParameterCount` attributes have a value supplied for their `Justification` argument.
```

Comment:
grammar: should be `...., which checks that`, rather than a new sentence

Reply:
Ah yes, that reads more clearly now